scripts/qa_crowbarsetup.sh

```diff
@@ -450,11 +450,11 @@ function iscloudver()
         local milestone=${v#*M}
         v=${v%M*}
         if [[ "$ver" -eq "$v" ]] && [[ $cloudsource =~ ^M[0-9]+$ ]] ; then
-            [ "${cloudsource#*M}" $operator "$milestone" ]
+            test "${cloudsource#*M}" $operator "$milestone"
```

Comment:
Hmm yeah, I would prefer that we fix `shellcheck` rather than switch to lots of `test` everywhere, which is much more old-fashioned style. Even old POSIX has `[` IIRC.